ProjFS.Mac/Scripts/Build.sh

```diff
@@ -12,7 +12,19 @@ PACKAGES=$ROOTDIR/packages
 
 PROJFS=$SRCDIR/ProjFS.Mac
 
-xcodebuild -configuration $CONFIGURATION -project $PROJFS/PrjFS.xcodeproj build -scheme 'Build All' -derivedDataPath $ROOTDIR/BuildOutput/ProjFS.Mac/Native || exit 1
+xcodebuild -configuration $CONFIGURATION -project $PROJFS/PrjFS.xcodeproj  -scheme 'Build All' -derivedDataPath $ROOTDIR/BuildOutput/ProjFS.Mac/Native build || exit 1
+
+if !(gem list --local | grep xcpretty); then
+   sudo gem install xcpretty
+fi
+
+# Run Tests and put output into a xml file
+xcodebuild -configuration $CONFIGURATION -project $PROJFS/PrjFS.xcodeproj -scheme 'Build All' test | xcpretty -r junit --output $PROJFS/TestResultJunit.xml
```

Comment:
@jrbriggs updated the build so the failed test appear in the Test category properly.

Reply:
@jeschu1 thanks! can you look at why the functional test run isn't working the same way?

Reply:
@jeschu1 that's a windows build -- can you double-check?

Reply:
https://dev.azure.com/mseng/AzureDevOps/_build/results?buildId=8278228&view=ms.vss-test-web.build-test-results-tab correct link :-)

Reply:
@jrbriggs apologies bad link.  I purposely broke the build so you have look now.  Check out the failures.